tests/send1.c

```diff
@@ -1211,6 +1211,73 @@ static void sendto_channel_opmod__local(void)
 	is_client_sendq_empty(local_chan_d, "Deaf; " MSG);
 	is_client_sendq(":" TEST_ME_ID "90004 TEST =" TEST_CHANNEL " :Hello World!" CRLF, server, MSG);
 	is_client_sendq_empty(server2, "No users to receive message; " MSG);
+
+	standard_free();
+}
+
+static void sendto_channel_opmod_statusmsg__local(void)
+{
+	standard_init();
+	ConfigChannel.opmod_send_statusmsg = true;
+
+	// This function does not support TS5...
+	standard_ids();
+
+	// Without CAP_CHW | CAP_EOPMOD
+	standard_server_caps(0, CAP_CHW | CAP_EOPMOD);
+
+	sendto_channel_opmod(local_chan_p, local_chan_p, channel, "TEST", "Hello World!");
+	is_client_sendq_empty(user, "Not on channel; " MSG);
+	is_client_sendq(":LChanPeon" TEST_ID_SUFFIX " TEST @" TEST_CHANNEL " :Hello World!" CRLF, local_chan_o, "On channel; " MSG);
+	is_client_sendq(":LChanPeon" TEST_ID_SUFFIX " TEST @" TEST_CHANNEL " :Hello World!" CRLF, local_chan_ov, "On channel; " MSG);
+	is_client_sendq_empty(local_chan_v, "Not +o; " MSG);
+	is_client_sendq_empty(local_chan_p, "Message source; " MSG);
+	is_client_sendq_empty(local_chan_d, "Deaf; " MSG);
+	is_client_sendq_empty(server, "No users to receive message; " MSG);
+	is_client_sendq_empty(server2, "No users to receive message; " MSG);
+
+	// With CAP_CHW, without CAP_EOPMOD
+	standard_server_caps(CAP_CHW, CAP_EOPMOD);
+
+	sendto_channel_opmod(local_chan_p, local_chan_p, channel, "TEST", "Hello World!");
+	is_client_sendq_empty(user, "Not on channel; " MSG);
+	is_client_sendq(":LChanPeon" TEST_ID_SUFFIX " TEST @" TEST_CHANNEL " :Hello World!" CRLF, local_chan_o, "On channel; " MSG);
+	is_client_sendq(":LChanPeon" TEST_ID_SUFFIX " TEST @" TEST_CHANNEL " :Hello World!" CRLF, local_chan_ov, "On channel; " MSG);
+	is_client_sendq_empty(local_chan_v, "Not +o; " MSG);
+	is_client_sendq_empty(local_chan_p, "Message source; " MSG);
+	is_client_sendq_empty(local_chan_d, "Deaf; " MSG);
+	is_client_sendq(":" TEST_ME_ID " NOTICE @" TEST_CHANNEL " :<LChanPeon:#test> Hello World!" CRLF, server, MSG);
```

Comment:
This hardcodes `#test` in the string, when it could use the `TEST_CHANNEL` macro like elsewhere?

Reply:
I just copied the existing tests. Guess I'll change those too.